tests/eas/generic.py

```diff
@@ -29,8 +29,6 @@
 from unittest import SkipTest
 
 import matplotlib.pyplot as plt
-import numpy as np
-import pylab as pl
 import os
```

Comment:
Since you are at it you could do a general cleanup of the imports, which means having something like:

```python
import one
import two
# etc.

from foo import bar
# etc.
```

Reply:
Done !